settings.gradle.kts

```diff
@@ -66,3 +67,5 @@ project(":spark-connector:spark-runtime-3.5").projectDir = file("spark-connector
 include("web")
 include("docs")
 include("integration-test-common")
+include("flink-connector:flink")
```

Comment:
seems these two lines are not necessary? it will include flink module if scalaVersion is 2.13